app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,49 @@
+footer {
+  display: flex;
+  align-items: center;
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  height: 200px;
+  border-radius: 15px;
+  margin: 0 auto;
+  margin-bottom: 15px;
+  padding: 15px;
+
+  .icon{
+    margin: 0 auto;
+    color: black;
+
+    &:hover {
+      color: #55679b;
+    }
+  }
```

Comment:
Can you change the icon color to $main-grey and the over color to $gold to fit the current design ?